halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/plugins/Plugin.java

```diff
@@ -27,32 +27,40 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import lombok.AccessLevel;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.Getter;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.Constructor;
 import org.yaml.snakeyaml.representer.Representer;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class Plugin extends Node {
-  public String name;
-  public Boolean enabled;
-  public String manifestLocation;
-  public Map<String, Object> options = new HashMap<>();
+  private String name;
+  private Boolean enabled;
+  private String manifestLocation;
+  private Map<String, Object> options = new HashMap<>();
+
+  @Getter(AccessLevel.NONE)
+  private Manifest manifest;
 
   @Override
   public String getNodeName() {
     return name;
   }
 
-  public Manifest generateManifest() {
+  public Manifest getManifest() {
```

Comment:
this change is related to downloading plugin manifests once. Since this is not necessary, lets skip doing this. Also, this is kind of ok if you spin up a new instance of halyard each time you want to deploy, but if it's a long running halyard process, downloading the manifest once will lead to stale manifests.